tests/apps/orderform/conftest.py

```diff
@@ -125,3 +136,152 @@ def fixture_mip_rna_orderform_sample() -> dict:
         "UDF/tumour purity": "5",
         "UDF/Comment": "other Elution buffer",
     }
+
+
+# Orderform fixtures
+
+
+@pytest.fixture(scope="session")
+def microbial_orderform(orderforms: Path) -> str:
+    """Orderform fixture for microbial samples"""
+    return Path(
+        orderforms
+        / f"{Orderform.MICROSALT}.{ORDERFORM_VERSIONS[Orderform.MICROSALT]}.microbial.xlsx"
+    ).as_posix()
+
+
+@pytest.fixture
+def sarscov2_orderform(orderforms: Path) -> str:
+    """Orderform fixture for sarscov2 samples"""
+    return Path(
+        orderforms
+        / f"{Orderform.SARS_COV_2}.{ORDERFORM_VERSIONS[Orderform.SARS_COV_2]}.sarscov2.xlsx"
+    ).as_posix()
+
+
+@pytest.fixture(scope="session")
+def rml_orderform(orderforms: Path) -> str:
+    """Orderform fixture for RML samples"""
+    return Path(
+        orderforms / f"{Orderform.RML}.{ORDERFORM_VERSIONS[Orderform.RML]}.rml.xlsx"
+    ).as_posix()
+
+
+@pytest.fixture(scope="session")
+def mip_json_orderform(orderforms: Path) -> dict:
+    """Load an example of json scout order."""
+    return json.load(open(orderforms / "mip.json"))
```

Comment:
```suggestion
    return json.load(open(Path(orderforms,"mip.json")))
```
Ideally you shouls never hard code the "/" but let Path handle it automatically. it is more future proof. Also for the rest of the fixtures.

I am rewriring the json io in another cg PR, but i will handle the merge conflict there 😄

Reply:
Fixed! Good luck with the conflict 😅